4_processing_features/extraction_utils.py

```diff
@@ -0,0 +1,159 @@
+import pathlib
```

Comment:
Consider adding docstring to the top of this file to clarify what this file is for.

Reply:
Makes sense to add that documentation, added now!